platform/android/export/export_plugin.cpp

```diff
@@ -2230,45 +2232,70 @@ bool EditorExportPlatformAndroid::can_export(const Ref<EditorExportPreset> &p_pr
-			err += TTR("\"Hand Tracking\" is only valid when \"Xr Mode\" is \"Oculus Mobile VrApi\" or \"OpenXR\".");
+			err += TTR("\"Hand Tracking\" is only valid when \"XR Mode\" is \"Oculus Mobile VrApi\" or \"OpenXR\".");
 			err += "\n";
 		}
 	}
 
 	if (xr_mode_index != XR_MODE_OPENXR) {
 		if (passthrough_mode > XR_PASSTHROUGH_NONE) {
 			valid = false;
-			err += TTR("\"Passthrough\" is only valid when \"Xr Mode\" is \"OpenXR\".");
+			err += TTR("\"Passthrough\" is only valid when \"XR Mode\" is \"OpenXR\".");
 			err += "\n";
 		}
 	}
 
-	if (int(p_preset->get("custom_template/export_format")) == EXPORT_FORMAT_AAB &&
+	if (int(p_preset->get("custom_build/export_format")) == EXPORT_FORMAT_AAB &&
 			!custom_build_enabled) {
 		valid = false;
 		err += TTR("\"Export AAB\" is only valid when \"Use Custom Build\" is enabled.");
 		err += "\n";
 	}
 
-	// Check the min sdk version
-	int min_sdk_version = p_preset->get("version/min_sdk");
-	if (min_sdk_version != DEFAULT_MIN_SDK_VERSION && !custom_build_enabled) {
-		valid = false;
-		err += TTR("Changing the \"Min Sdk\" is only valid when \"Use Custom Build\" is enabled.");
-		err += "\n";
+	// Check the min sdk version.
+	String min_sdk_str = p_preset->get("custom_build/min_sdk");
+	int min_sdk_int = DEFAULT_MIN_SDK_VERSION;
+	if (!min_sdk_str.empty()) { // Empty means no override, nothing to do.
+		if (!custom_build_enabled) {
+			valid = false;
+			err += TTR("\"Min SDK\" can only be overridden when \"Use Custom Build\" is enabled.");
+			err += "\n";
+		}
+		if (!min_sdk_str.is_valid_integer()) {
+			valid = false;
+			err += vformat(TTR("\"Min SDK\" should be a valid integer, but got \"%s\" which is invalid."), min_sdk_str);
+			err += "\n";
+		} else {
+			min_sdk_int = min_sdk_str.to_int();
+			if (min_sdk_int < DEFAULT_MIN_SDK_VERSION) {
+				valid = false;
+				err += vformat(TTR("\"Min SDK\" cannot be lower than %d, which is the version needed by the Godot library."), DEFAULT_MIN_SDK_VERSION);
+				err += "\n";
+			}
+		}
 	}
 
-	// Check the target sdk version
-	int target_sdk_version = p_preset->get("version/target_sdk");
-	if (target_sdk_version != DEFAULT_TARGET_SDK_VERSION && !custom_build_enabled) {
-		valid = false;
-		err += TTR("Changing the \"Target Sdk\" is only valid when \"Use Custom Build\" is enabled.");
-		err += "\n";
+	// Check the target sdk version.
+	String target_sdk_str = p_preset->get("custom_build/target_sdk");
+	int target_sdk_int = DEFAULT_TARGET_SDK_VERSION;
+	if (!target_sdk_str.empty()) { // Empty means no override, nothing to do.
+		if (!custom_build_enabled) {
+			valid = false;
+			err += TTR("\"Target SDK\" can only be overridden when \"Use Custom Build\" is enabled.");
+			err += "\n";
+		}
+		if (!target_sdk_str.is_valid_integer()) {
+			valid = false;
+			err += vformat(TTR("\"Target SDK\" should be a valid integer, but got \"%s\" which is invalid."), target_sdk_str);
+			err += "\n";
+		} else {
+			target_sdk_int = target_sdk_str.to_int();
+		}
```

Comment:
Same comment as in https://github.com/godotengine/godot/pull/62611/files#r912493588, we should include a warning when the target sdk is greater than the default.

Reply:
Fixed.